modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -25,10 +25,26 @@ an {product-title} `ConfigMap` object containing the CA. This is done by running
 +
 [source,terminal]
 ----
-$ oc create configmap ca-config-map --from-file=ca.crt=/path/to/ca -n openshift-config
+$ oc create configmap ca-config-map --from-file=ca.crt=/path/to/ca -n openshift-config <1>
 ----
+<1> The CA must be stored in the `ca.crt` key of the `ConfigMap` object.
 +
-The CA must be stored in the `ca.crt` key of the `ConfigMap` object.
+[TIP]
+====
+You can alternatively apply the following YAML to create the config map:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ca-config-map
+  namespace: openshift-config
+type: Opaque
+data:
+  ca.crt: <base64_encoded_CA_certificate_PEM>
```

Comment:
you can use the multi-line PIPE notation here as well

Reply:
Also, IIRC the PEM is not base64-encoded here, that might have been a mistake from my side.

Reply:
@stlaz You're right. Updated the replaceable value to just <CA_certificate_PEM> if that works